programs/perpetuals/tests/test.rs

```diff
@@ -0,0 +1,14 @@
+use solana_program_test::tokio;
```

Comment:
maybe rename this file to `main.rs`? too many tests, not clear how this is different